frontend/src/citizen-frontend/styled.d.ts

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import 'styled-components'
+import { Theme } from 'lib-common/theme'
+
+declare module 'styled-components' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  export interface DefaultTheme extends Theme {}
```

Comment:
Check styled-component doc https://styled-components.com/docs/api#typescript

Reply:
And specially line: "DefaultTheme is being used as an interface of props.theme out of the box. By default the interface DefaultTheme is empty so that's why we need to extend it."